### PR TITLE
docs: Updated integration docs with required package installations

### PR DESCRIPTION
**Pull request by NithinBairapaka**
**Title:** Updated integration docs with required package installations
   **Issue:**  #22005

### CODE REVIEW

File: docs/docs/integrations/llms/ai21.ipynb
Comment (efriis) [inline docs/docs/integrations/llms/ai21.ipynb:1]: Could you update to not mention core unless it's *only* core? Core automatically comes with all partner packages, e.g. langchain-ai21

Comment (NithinBairapaka) [reply docs/docs/integrations/llms/ai21.ipynb:1]: Hi efriis, Thank you for your suggestion. I have updated the files accordingly.

Comment (baskaryan) [reply docs/docs/integrations/llms/ai21.ipynb:1]: still seeing few pages that install langchain-core, shouldn't be needed in any of these

Comment (NithinBairapaka) [reply docs/docs/integrations/llms/ai21.ipynb:1]: Hi baskaryan, resolved the issue.

**Review by efriis — APPROVED** [review]

**Pull request merged**